feat(collaboration): [OCISDEV-336] support disabling editors by extensions (#11750)
owncloud/ocis: services/collaboration/README.md

* `COLLABORATION_WOPI_ENABLE_MOBILE`:\
  This option enables the mobile view for the Office Web App and only applies to OnlyOffice. While OnlyOffice `ce` supports mobile view mode only, the `ee` and `de` versions support mobile view in both view and edit modes. Set the product edition accordingly via `COLLABORATION_APP_PRODUCT_EDITION`.

* `COLLABORATION_WOPI_DISABLED_EXTENSIONS`:\

>   List of extensions to disable. If an extension is disabled, it will not be available to the office web frontend.

Reviewer: ```suggestion
  List of extensions to disable: Disabling an extension will make it unavailable to the Office web front end.
```
